nrf_modem/doc/sockets.rst

```diff
@@ -298,6 +300,9 @@ NRF_SO_SEC_DTLS_CID_STATUS
 
 NRF_SO_SEC_DTLS_CONN_SAVE
    Save DTLS connection.
+   This socket option can be used to pause a session that is not frequently used by the application. Saving the session will free memory in the modem that can be used for other connections.
```

Comment:
```suggestion
   This socket option can be used to pause a session that is not frequently used by the application.
   Saving the session will free memory in the modem, so the memory can be used for other connections.
```
I think it's supposed to say that the memory is what can be used for other connections. If it's the modem that can be used for other connections, apply only the line break.